### ISSUE TITLE
Web browser extensions

### ISSUE THREAD

**Issue by ZM71** [AndreyPavlenko/Fermata#569]
Extensions not working the browser does not specify what app it is to use for the extentions would like to have support for Ubloxk Origin and I Don't Care About Cookies.

**Comment by AndreyPavlenko** [comment]
Because you can't use any extension, implemented for other apps.

**Comment by ZM71** [comment]
Ok but the browser doesn't support extensions at all?

**Comment by AndreyPavlenko** [comment]
No. You can't just get an extension, developed for one browser and install on another.

**Comment by ZM71** [comment]
Ok could you implement that feature?

**Comment by AndreyPavlenko** [comment]
Which one?

**Comment by ZM71** [comment]
Ublock Origin and I Don't Care About Cookies would be nice 👍.
